tests/models/udop/test_processor_udop.py

```diff
@@ -56,38 +56,45 @@ class UdopProcessorTest(ProcessorTesterMixin, unittest.TestCase):
     processor_class = UdopProcessor
     maxDiff = None
 
-    def setUp(self):
-        self.tmpdirname = tempfile.mkdtemp()
+    @classmethod
+    def setUpClass(cls):
+        cls.tmpdirname = tempfile.mkdtemp()
+        cls.addClassCleanup(lambda tempdir=cls.tmpdirname: shutil.rmtree(tempdir))
```

Comment:
IMO, we keep either `cls.addClassCleanup` or `tearDownClass` but not both.

Reply:
Moved to only using `tearDownClass()`!